spec/text-document-spec.coffee

```diff
@@ -2,28 +2,36 @@ Point = require "../src/point"
 TextDocument = require "../src/text-document"
 
 describe "TextDocument", ->
-  document = null
```

Comment:
To clarify, I mean we add new `characterIndexForPosition` and `positionForCharacterIndex` methods and test in terms of them. This will make it easier to transition from the old `TextBuffer` to `TextDocument`.